src/assignments/2-operator.js

```diff
@@ -14,6 +14,26 @@ output example
 */
 // start here writing the code for program1
 
+const num1 = Number(prompt('Enter first number'));
+const num2 = Number(prompt('Enter second number'));
+
+let result;
+
+result = num1 + num2;
+console.log(`${num1}+${num2}=${result}`);
+
+result = num1 - num2;
+console.log(`${num1}-${num2}=${result}`);
+
+result = num1 * num2;
+console.log(`${num1}*${num2}=${result}`);
+
+result = num1 / num2;
+console.log(`${num1}/${num2}=${result.toPrecision(5)}`);
+
+result = num1 % num2;
```

Comment:
it is necessary to check if the num2 is bigger than 0 or not, then do the math calculation.